Less flickering

kitzberger opened the pull request.

Patta commented:
@kitzberger nice, I will test this as soon as possible. 

First question: 
Where does the integrator have to insert the viewhelper code mentioned in the readme?
In the powermail layout or page layout/template? Then the code would always be inserted, regardless of whether powermail_cond is used on the page or not. I think it would be better if the condition code is added directly with powermail_cond, perhaps with its own powermail_cond fluid layout/template, which does not have to be set separately by the integrator.

kitzberger commented:
@Patta, since EXT:powermail_cond doesn't come with any templates/partials overrides so far, I figured let's keep it that way. That's why (for this early stage of the patch) the integrator has to insert that snippet manually.

The code can be inserted into powermail's layout or the form template. Doesn't matter because it comes with an `id` attribute which will be targetted by the JS.

sbusemann commented:
https://projekte.in2code.de/issues/66407

pixeldesu reviewed (changes requested):
Thank you for your contribution!

From initial testing, this seems like a great addition, but I do have some remarks for the code that should get adjusted before we merge this into the project!

* Please remove debug outputs, this mainly concerns the `console.log` statements in the JavaScript code. If you need to describe the conditional branches, feel free to use comments instead!
* This is mainly a structuring/"meaning" thing, but I don't think that `ConditionUtility` is a _utility_. Technically, you moved the domain logic of the controller, including fetching of forms and conditions somewhere else, so it's probably more appropriate to name this `ConditionService` instead (and move it into a `Service` folder too)

Once these adjustments have been made, things should be ready to merge this!

kitzberger commented:
@pixeldesu, thanks for your feedback! I removed all console.log's that I've added, changed the utility's class name and rebase everything to the latest `develop` branch.

pixeldesu reviewed (approved):
Thanks! Tested it and works fine! 👍
